css/styles.css

```diff
@@ -561,47 +574,63 @@ Contains email and socials
   }
   */
 
+  header {
+    height: 8vh;
+  }
 
-    .headline,
-    .about-myself {
-      margin: 0;
-    }
+  header #union {
+    display: none;
+  }
 
-    .spacing-default {
-      margin: 2% auto;
-    }
+  header #navbar {
+    display: flex;
+  }
 
-    .arrow {
-      margin-bottom: 0;
-    }
+  .about-myself {
+    margin: 0;
+  }
 
-    .recent-work {
-      width: 92%;
-    }
+  .headline {
+    margin-top: 5%;
+  }
 
-    .recent-work-text,
-    .about-myself-text {
-      margin: 0 auto 5% auto;
-    }
+  .header-bg {
+    background-image: url(../images/Header-llustration-desktop.png);
+  }
+
+  .spacing-default {
+    margin: 2% auto;
+  }
+
+  .arrow {  /* TO CENTER */
+    margin: 0% auto 20% auto;
+  }
+
+  .recent-work {
+    width: 92%;
+  }
+
+  .recent-work-text,
+  .about-myself-text {
+    margin: 0 auto 5% auto;
```

Comment:
- [ ] The `My Recent Work` heading in the figma design template is `Capatalized` please make sure that your heading is `Capatalized`

Figma design:

<img width="502" alt="Screenshot 2023-03-31 at 9 27 29 AM" src="https://user-images.githubusercontent.com/44798044/229030681-5fa1a8d2-7864-4958-b0d3-9d9941fff184.png">

Your design:

<img width="655" alt="Screenshot 2023-03-31 at 9 27 38 AM" src="https://user-images.githubusercontent.com/44798044/229030719-0092d044-d541-45fa-8ae3-be86026d3b82.png">